index.html

```diff
@@ -55,8 +55,167 @@ <h2>I'm a Software Developer</h2>
         </li>
       </ul>
       <headLine class="bgImage">
-        <!-- <img  src="images\Illustration.png" /> -->
+        <!--Image for the background -->
       </headLine>
     </section>
+    <!-- Work Section -->
+    <section class="work">
+      <div class="recentWorks">
+        <h2>My Recent Works</h2>
+        <img src="images\spacer.svg" alt="Spacer" />
+      </div>
+      <div class="projects">
+        <img src="images/mainPH.svg" alt="img placeholder" />
+        <h2>Multi-Post Stories</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="tools">
+          <li><button class="toolsBtn">CSS</button></li>
+          <li><button class="toolsBtn">HTML</button></li>
+          <li><button class="toolsBtn">Bootstrap</button></li>
+          <li><button class="toolsBtn">Ruby</button></li>
+        </ul>
+        <button class="mainSeeBtn">See Project</button>
+      </div>
+      <!-- Card 1-->
+      <div class="cardContainer">
```

Comment:
- [ ] I kindly suggest you fix your cards from overflowing it's container. This is causing an horizontal scroll on the page.

<img width="390" alt="image" src="https://user-images.githubusercontent.com/52934128/219387203-4ed67e50-0ab8-4e8e-a187-b6ad30924202.png">